lib/containers/utils.pm

```diff
@@ -201,18 +201,16 @@ sub basic_container_tests {
 }
 
 # Setup environment
-sub container_set_up {
+sub prepare_img {
```

Comment:
```suggestion
sub setup_env {
```

Reply:
I disagree here - as I said, currently we have `prepare_img()`, `build_img()` and `test_img()` and I would like to keep that common suffix.

Reply:
ok it was just idea don't want to insist